doc/7/essentials/events/index.md

```diff
@@ -32,6 +32,22 @@ Triggered when Kuzzle discards a request, typically if no connection is establis
 
 Triggered when the current session has been unexpectedly disconnected.
 
+<SinceBadge version="auto-version"/>
+**Callback arguments:**
+
+`@param {object} context`
+
+| Property | Type              | Description                                |
+| -------- | ----------------- | ------------------------------------------ |
+| `origin` | <pre>string</pre> | Indicate what is causing the disconnection |
+
+**Origins**
+
+| Name                        | Description                                                                                                                                                        |
+| --------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| `websocket/auth-renewal`    | The websocket protocol si reconnecting to renew the token. See [Websocket Cookie Authentication](sdk/js/7/protocols/websocket/introduction#cookie-authentication). |
+| `network/connection-closed` | The disconnection is either manual or issued by Kuzzle                                                                                                             |
```

Comment:
We could have `user/connection-closed` instead since the action is not initiated by the network layer but by the user